deploy/cr.yaml

```diff
@@ -36,6 +36,18 @@ spec:
     size: 3
     image: percona/percona-xtradb-cluster:8.0.22-13.1
     autoRecovery: true
+#   expose:
```

Comment:
@qJkee  please add additional space after '#' all because now if you uncomment these lines you will have broken manifest

Reply:
757afff